All-units prebuilt header error on `pascal` on MSVC

When using https://aurora-opensource.github.io/au/0.3.5/au_all_units.hh with my C++20 project, I get these errors, only on the `pascal` definition. I was originally on a different commit version of the file, and after the change to 0.3.5 proper it had the exact same errors. If I rename `pascal` to `pascal2`, the error goes away. Maybe there's a shadowed type, keyword, or global here, somehow?

```log
Au\au_all_units.h(6087): error C2059: syntax error: '='
Au\au_all_units.h(6087): error C2143: syntax error: missing ';' before '{'
Au\au_all_units.h(6087): error C2447: '{': missing function header (old-style formal list?)
```

### Compile info:
- Windows 11
- RelWithDebInfo Configuration
- Toolchain: msvc_x64_64
- MSVC v143 - VS 2022 17.10.3
- cl version 14.40
- CMake build through Visual Studio's CMake support (Ninja)
- CMake configuration:
```cmake
set(CMAKE_CXX_STANDARD 20) 
set(CMAKE_CXX_STANDARD_REQUIRED ON)
set(CMAKE_CXX_EXTENSIONS OFF)
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

--- a/au/code/au/units/pascals.hh
+++ b/au/code/au/units/pascals.hh
@@ -33,7 +33,13 @@ constexpr const char PascalsLabel<T>::label[];
 struct Pascals : decltype(Newtons{} / squared(Meters{})), PascalsLabel<void> {
     using PascalsLabel<void>::label;
 };
-constexpr auto pascal = SingularNameFor<Pascals>{};
+
+#ifndef pascal
+[[deprecated(
+    "Conflicts with the `pascal` macro from <Windows.h>; declare manually "
+    "instead.")]] constexpr auto pascal = SingularNameFor<Pascals>{};
+#endif
+
 constexpr auto pascals = QuantityMaker<Pascals>{};
 constexpr QuantityPointMaker<Pascals> pascals_pt{};
 

